mirage/scenarios/default.js

```diff
@@ -532,275 +699,161 @@ export default function(server) {
+    displayDate2: null,
+  });
+
+  server.create('milestone', 'ceqrFeePayment', {
+    project: seedBPUserProjects[0],
+    statuscode: 'Completed',
+    actualstartdate: moment().subtract(70, 'days'),
+    displayDate: moment().subtract(70, 'days'),
+    actualenddate: null,
+    displayDate2: null,
+  });
+
+  server.create('milestone', 'filedEasReview', {
+    project: seedBPUserProjects[0],
+    statuscode: 'Completed',
+    actualstartdate: moment().subtract(40, 'days'),
+    displayDate: moment().subtract(40, 'days'),
+    actualenddate: null,
+    displayDate2: null,
+  });
+
+  server.create('milestone', 'eisPublicScopingMeeting', {
+    project: seedBPUserProjects[0],
+    statuscode: 'Completed',
+    actualstartdate: moment().subtract(30, 'days'),
+    displayDate: moment().subtract(30, 'days'),
+    actualenddate: null,
+    displayDate2: null,
+  });
+
+  server.create('milestone', 'finalScopeOfWorkIssued', {
+    project: seedBPUserProjects[0],
+    statuscode: 'Completed',
+    actualstartdate: moment().subtract(30, 'days'),
+    displayDate: moment().subtract(30, 'days'),
+    actualenddate: null,
+    displayDate2: null,
+  });
+
+  server.create('milestone', 'nocOfDraftIssued', {
+    project: seedBPUserProjects[0],
+    statuscode: 'Completed',
+    actualstartdate: moment().subtract(5, 'days'),
+    displayDate: moment().subtract(5, 'days'),
+    actualenddate: null,
+    displayDate2: null,
+  });
+
+  server.create('milestone', 'deisPublicHearingHeld', {
+    project: seedBPUserProjects[0],
+    statuscode: 'Completed',
+    actualstartdate: moment().subtract(20, 'days'),
+    displayDate: moment().subtract(20, 'days'),
+    actualenddate: null,
+    displayDate2: null,
+  });
+
+  server.create('milestone', 'feisPublicSubmittedAndReview', {
+    project: seedBPUserProjects[0],
+    statuscode: 'Completed',
+    actualstartdate: moment().subtract(10, 'days'),
+    displayDate: moment().subtract(10, 'days'),
+    actualenddate: null,
+    displayDate2: null,
+  });
+
+  server.create('milestone', 'certifiedReferred', {
+    project: seedBPUserProjects[0],
+    statuscode: 'Not Started',
+    actualstartdate: moment().subtract(2, 'days'),
+    displayDate: moment().subtract(2, 'days'),
+    actualenddate: null,
+    displayDate2: null,
+  });
+
+  // Here, mock that the BP user's first project
+  // has reached Public Review but not yet their respective Review Period
+  server.create('milestone', 'communityBoardReview', {
+    project: seedBPUserProjects[0],
+    statuscode: 'In Progress',
+    actualstartdate: moment().subtract(1, 'days'),
+    displayDate: moment().subtract(1, 'days'),
+    actualenddate: null,
+    displayDate2: null,
+  });
+
+  server.create('milestone', 'boroughPresidentReview', {
+    project: seedBPUserProjects[0],
+    statuscode: 'Not Started',
+    actualstartdate: moment().add(19, 'days'),
+    displayDate: moment().add(19, 'days'),
+    actualenddate: null,
+    displayDate2: null,
+  });
```

Comment:
This scenario file is getting really long. Do you think this would be better addressed with Mirage fixtures? -> https://www.ember-cli-mirage.com/docs/data-layer/fixtures

Reply:
yes! I had the same feeling, and was wondering if fixtures would help. I'll open an issue to look into it.